### DISCUSSION TITLE
DiceBear 7.0 - Release Notes

### DISCUSSION THREAD

**Discussion by FlorianKoerner** [dicebear/dicebear#351] [Announcements]
## New Features:
<ul>
<li>
<b>Avatar style "rings"</b> (Licensed under CC0 1.0):<br />
  Introducing a brand new avatar style called <a href="https://www.dicebear.com/styles/rings">rings</a>, which gives your avatars a unique and captivating look. With this addition, you have even more creative freedom to create different avatars to suit the needs of your application.

<p>
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Mimi" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Sasha" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Lilly" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Tigger" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Bella" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Zoe" width="48" />
  <img src="https://api.dicebear.com/7.x/rings/svg?seed=Kitty" width="48" />
</p>
</li>
<li>
<b>HTTP API Endpoint for Avatar Style JSON Schema</b>:<br />
  In response to user feedback (#337, #341), we have implemented an HTTP API endpoint that provides the JSON schema for each avatar style. You can access this endpoint using the following URL pattern: <code>https://api.dicebear.com/7.x/<style>/schema.json</code>.
</li>
</ul>

## Bugfixes:
- **Fixed probability calculation** (#315):
  We've fixed a problem with the likelihood calculation. Now when the likelihood is set to zero, the library will consistently return `false` as expected.

As always, we appreciate your support and feedback.

<hr /><em>This discussion was created from the release <a href='https://github.com/dicebear/dicebear/releases/tag/v7.0.0'>DiceBear 7.0 - Release Notes</a>.</em>

**Comment by CraigVG** [comment]
Including the schema is a lovely change, thank you!
